main.pm

```diff
@@ -857,6 +857,14 @@ else {
     }
     elsif (get_var("BOOT_HDD_IMAGE")) {
         loadtest "boot/boot_to_desktop.pm";
+        if (get_var("ISCSI_SERVER")) {
+            set_var('INSTALLONLY', 1);
```

Comment:
I agree but how do you track ideas like this? Do you rely on others to grab your suggestion and go ahead and if nothing happens you do it or if nobody cares it wil be forgotten and it might not have been that important after all?